tests/sentry/api/endpoints/test_group_events.py

```diff
@@ -228,3 +231,27 @@ def test_environment(self):
 
         assert response.status_code == 200, response.content
         assert response.data == []
+
+    def test_filters_based_on_retention(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+        group = self.create_group(project=project)
+        self.create_event(
+            'a' *
+            32,
```

Comment:
aside: My god, is this autopep8?

Reply:
hrm i feel like it must be wtf why would it do that